.travis.yml

```diff
@@ -15,7 +15,9 @@ cache:
     - docs/node_modules
 before_script: npm prune
 
-script: npm run test-with-coverage
+script:
+  - npm run flow:check
+  - npm run test-with-coverage
```

Comment:
Ok. That should be easy. But I will wait for fix/windows branch to be merged before implementing this to avoid conflict (most of code was written overthere)